listenbrainz/webserver/templates/index/index.html

```diff
@@ -30,6 +31,9 @@ <h2 class="page-title">Welcome to ListenBrainz!</h2>
       <p>
         Need help? Ask our <a href="https://community.metabrainz.org/c/listenbrainz">community</a>!
       </p>
+    </div>
+
```

Comment:
I think it would be good to add a small section here "Discover new music".

Throwing some words down in case it helps:

Based on your listening history we recommend music you might like, new music you missed out on 
 from your favorite artists, as well as what similar users have been listening to.
There are many ways to explore and discover new music with ListenBrainz!

Reply:
Done!